### PR TITLE
feat: added new website

### PR DESCRIPTION
**Pull request by calebjclark**

### CODE REVIEW

File: website/src/lib/Data.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+export interface IPage {
+  content: string;
+  title: string;
+  subtitles: { value: string, depth: number, anchor: string }[];
+}
+
+const repoLinks: { [key: string]: string } = {
+  hero: 'https://github.com/ulixee/hero/tree/main/docs/main',
+  databox: 'https://github.com/ulixee/hero/tree/main/docs/main',
```
Comment (blakebyrnes) [inline website/src/lib/Data.ts:19]: links are wrong

**Review by blakebyrnes — COMMENTED** [review]
Is the docs project with grid some configs in it supposed to still be in the root?

File: website/src/lib/Data.ts (hunk 1 above)
Comment (calebjclark) [reply website/src/lib/Data.ts]: Fixed

**Comment by calebjclark** [comment]
Removed the old docs.

**Pull request merged**